### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by KevinEtienne**

### CODE REVIEW

**Comment by KevinEtienne** [comment]
@adam-thomas can you review/merge please?

File: setup.cfg [hunk 1]
```diff
@@ -2,7 +2,7 @@
 max-line-length = 90
 max-complexity = 10
 exclude = *migrations*
-ignore = D100,D203,D204
+ignore = D100,D104,D203,D204
 statistics=true
 application-import-names = pgcrypto
-import-order-style = google
+import-order-style = smarkets
```
Comment (meshy) [inline setup.cfg]: This is new on me! ... and awesome! Thanks!

Comment (KevinEtienne) [inline setup.cfg]: It has been added in https://github.com/public/flake8-import-order/pull/57

> smarkets - style as google only with import statements before from X import

I'm happy they've made the move to keep compability

**Comment by adam-thomas** [comment]
:+1:

**Pull request merged**